testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppByUrlActivity.kt

```diff
@@ -25,9 +25,9 @@ class MiniAppInputActivity : BaseActivity(), PreloadMiniAppWindow.PreloadMiniApp
     private val preloadMiniAppWindow by lazy { PreloadMiniAppWindow(this, this) }
 
     sealed class InputDisplay(val input: String) {
-        class AppId(input: String): InputDisplay(input)
-        class Url(input: String): InputDisplay(input)
-        class None: InputDisplay("")
+        class AppId(input: String) : InputDisplay(input)
```

Comment:
This one (speaking of functionality) should probably also be removed (maybe here's the confusion) the MiniAppByUrlActivity should be used for local testing with eg `http://localhost:1337` but not load a miniapp by it's id. The list search should already provide a way to input an id to find the MiniApp. Can you confirm this is the intention behind `InputDisplay.AppId`?

Reply:
Can confirm, the page is intended to have not by the **URL** but also the **appId**, to begin with. @laubengaier.

Reply:
```
if (URLUtil.isValidUrl(input)) {
    onValidUI()
    InputDisplay.Url(input)
} else if (!AppSettings.instance.isDisplayUrlPreviewMode && !input.isInvalidUuid()) {
    onValidUI()
    InputDisplay.AppId(input)
}
```

which can be toggled in the settings page/

Reply:
oh yeah then please remove it (also the settings option) we only need the url functionality here as you can see here
https://rak.box.com/s/noqj511lvpq29rgg5qouc0jjvp2dmush
(normally it should load the miniapp if it's running locally instead of the charles warning)